src/org/javarosa/core/model/condition/EvaluationContext.java

```diff
@@ -116,8 +116,12 @@ public void setOriginalContext(TreeReference ref) {
 	}
 
 	public TreeReference getOriginalContext() {
-		if (this.original == null) { return this.contextNode; }
-		else { return this.original; }
+		if (this.original == null) {
```

Comment:
As above it's only `LeftCurly` checkstyle. I don't care about anything else here.